### DISCUSSION TITLE
Area Chart dotted background

### DISCUSSION THREAD

**Discussion by Flowko** [f5/unovis#382] [Q&A] [answered]
i woud love to know how we can achieve something like this
![image](https://github.com/f5/unovis/assets/35883748/cad970b6-1b3a-443f-af36-d85ce7fa776c)

i can set a div behind it, but that will be applied to the whole chart with its axis and all, but only want it to be behind the area component.

not sure if there's a way to do that! 

thank you!

**Comment by rokotyan** [comment, marked as answer]
Hi @Flowko, we don't currently support dotted gridlines and it's not in our plans.

However, what you can do, is:
1. Specify the chart margins explicitly by setting `autoMargin` to `false`
2. Provide your exact margin values (e.g. `margin: { top: 10, bottom: 60, left: 125, right: 10 }`). This will pin your axes location.
3. Position your div behind the chart accordingly.

**Reply by Flowko** [reply]
Yeah, was hoping there might be a param or something, but thank you, this will do just fine
